### PR TITLE
Prepare for release of analyzer, bundler, polyserve and CLI.

### PR DESCRIPTION
**Pull request by usergenic**
- [ ] polymer-analyzer@3.2.2
- [ ] polymer-bundler@4.0.5
- [ ] polyserve@0.27.15
- [ ] polymer-cli@1.9.5

### CODE REVIEW

File: packages/analyzer/CHANGELOG.md [hunk 1]
```diff
@@ -5,10 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
-* Removed non-essential files from published package, such as tests.
+<!-- ## Unreleased -->
 <!-- Add new, unreleased changes here. -->
 
+## [3.2.2] - 2018-01-08
```
Comment (aomarks) [inline packages/analyzer/CHANGELOG.md]: 2019-01-10

File: packages/bundler/CHANGELOG.md [hunk 2]
```diff
@@ -5,11 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+<!-- ## Unreleased -->
+<!-- Add new, unreleased changes here. -->
+
+## 4.0.5 - 2019-01-08
```
Comment (aomarks) [inline packages/bundler/CHANGELOG.md]: 2019-01-10

File: packages/cli/CHANGELOG.md [hunk 3]
```diff
@@ -5,10 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
-* Add `--help` description for `--version`.
+<!-- ## Unreleased -->
 <!-- Add new, unreleased changes here. -->
 
+## v1.9.5 [01-08-2019]
```
Comment (aomarks) [inline packages/cli/CHANGELOG.md]: 01-10-2019

File: packages/polyserve/CHANGELOG.md [hunk 4]
```diff
@@ -5,10 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
-* Add `--allow-origin` flag which sets the `Access-Control-Allow-Origin` header.
+<!-- ## Unreleased -->
 <!-- Add new, unreleased items here. -->
 
+## [0.27.15] (2019-01-08)
```
Comment (aomarks) [inline packages/polyserve/CHANGELOG.md]: 2019-01-10

File: packages/analyzer/CHANGELOG.md (hunk 1 above)
Comment (usergenic) [reply packages/analyzer/CHANGELOG.md]: Thanks. This is going to happen a lot to me I think.

**Pull request merged**